shared/ui/testing/src/main/java/com/ivy/ui/testing/IvyPreviews.kt

```diff
@@ -0,0 +1,20 @@
+package com.ivy.ui.testing
+
+import androidx.compose.ui.tooling.preview.Preview
+
+@Retention(AnnotationRetention.BINARY)
+@Target(
+    AnnotationTarget.ANNOTATION_CLASS,
+    AnnotationTarget.FUNCTION
+)
+@Preview(
+    apiLevel = 34,
+    name = "Medium Phone",
+    device = "spec:id=reference_phone,shape=Normal,width=411,height=891,unit=dp,dpi=420",
+)
+@Preview(
+    apiLevel = 34,
+    name = "Large Phone",
+    device = "id:pixel_8_pro",
+)
+annotation class IvyPreviews
```

Comment:
Does it test both Light & Dark mode?

Reply:
This annotation is not related to dark mode or light mode; it is about my need (multipreview) to export images on both small-sized and large-sized mobile devices

Reply:
Can you create a single annotation that does both:
- multi screen
- light & dark

Most bugs happen on small screens or UIs lacking dark mode support

Reply:
@ILIYANGERMANOV I don't think we can create a annotation can show all dark and light mode with @Preview.(because dark background is not produced by the @Preview annotation)

And currently in screenshot test function, i already write the function export the UI in both dark and light mode. 

<img width="497" alt="image" src="https://github.com/user-attachments/assets/d82ba5ad-bd9b-46a2-99d6-695e4ce57a89">

During the development process, dev still needs to change (theme paramater) in the preview function to check for sure

<img width="621" alt="image" src="https://github.com/user-attachments/assets/d1962148-7c9c-479e-a229-c0343ea29b89">

Btw, which module should i put the definitation of @IvyPreviews, maybe :shared:ui:core, i guess ? wdyt ?

Reply:
Hey @isaacnguyen0809 good questions!
1. Put it in `shared:ui:core`
2. Create an `IvyPreview` composable function with `content @Composable () -> Unit` slot if such one doesn't exist

The dev writing what you shown is fine 👍

Reply:
1. It's done
2. It already exists. But some screens keep using `IvyWalletPreview` because they're using a composable function in `BoxWithConstraintScope`, and `IvyPreview` will not work.

<img width="705" alt="image" src="https://github.com/user-attachments/assets/e3d9b80c-5380-4407-980a-3019f822b66a">